Arelle/EdgarRenderer
# Edgar 22.2.2 updates

### Reason for change

 * Redline removal of unneeded <span> element was removing unremovable elements, like `<tr>`.
 * ix viewer production.min.js was not latest in use by EDGAR, when using prior version on [this filing](https://www.sec.gov/Archives/edgar/data/1512931/000141057822002020/0001410578-22-002020-index.htm) the ix viewer sections menu had superfluous sections of "Prospectus" and "RR Summaries" with spinning wait icons. 

### Description of change

 * Only remove `<span>` element when removal of redline style property leaves it without any other 
 * Update production.min.js to latest in use by EDGAR

### Steps to Test

 * Use test case with a redline style property (such as efm\conf\525-ix-syntax\efm\16-redlining\i12010gd) and verify that a `<span>` becoming barren without the redline property is properly removed but a `<tr>` or other is left intact.
 * On [this filing, downloaded and run from local files](https://www.sec.gov/Archives/edgar/data/1512931/000141057822002020/0001410578-22-002020-index.htm) ensure that the ix viewer sections menu has only "Document and Entity Information" and "Notes to the Financial Statements" and no "Prospectus" or "RR Summaries" entries.

review:
https://github.com/orgs/Arelle/teams/arelle
